migrations/20240404084539_nationalityupdate_again.up.sql

```diff
@@ -0,0 +1,461 @@
+    ('23', 'Zaporizhzhia Oblast', 'UA'),
+    ('26', 'Ivano-Frankivsk Oblast', 'UA'),
+    ('30', 'Kyiv', 'UA'),
+    ('32', 'Kyiv Oblast', 'UA'),
+    ('35', 'Kirovohrad Oblast', 'UA'),
+    ('40', 'Sevastopol', 'UA'),
+    ('43', 'Autonomous Republic of Crimea', 'UA'),
+    ('46', 'Lviv Oblast', 'UA'),
+    ('48', 'Mykolaiv Oblast', 'UA'),
+    ('51', 'Odesa Oblast', 'UA'),
+    ('53', 'Poltava Oblast', 'UA'),
+    ('56', 'Rivne Oblast', 'UA'),
+    ('59', 'Sumy Oblast', 'UA'),
+    ('61', 'Ternopil Oblast', 'UA'),
+    ('63', 'Kharkiv Oblast', 'UA'),
+    ('65', 'Kherson Oblast', 'UA'),
+    ('68', 'Khmelnytskyi Oblast', 'UA'),
+    ('71', 'Cherkasy Oblast', 'UA'),
+    ('74', 'Chernihiv Oblast', 'UA'),
+    ('77', 'Chernivtsi Oblast', 'UA'),
+    ('01', 'Ostfold', 'NO'),
+    ('02', 'Akershus', 'NO'),
+    ('03', 'Oslo', 'NO'),
+    ('06', 'Buskerud', 'NO'),
+    ('07', 'Vestfold', 'NO'),
+    ('08', 'Telemark', 'NO'),
+    ('11', 'Rogaland', 'NO'),
+    ('15', 'More and Romsdal', 'NO'),
+    ('18', 'Northland', 'NO'),
+    ('19', 'Troms', 'NO'),
+    ('20', 'Finnmark', 'NO'),
+    ('34', 'Innlandet', 'NO'),
+    ('42', 'Agder', 'NO'),
+    ('46', 'Vestland', 'NO'),
+    ('50', 'Trondelag', 'NO'),
+    ('SJM', 'Svalbard and Jan Malen', 'NO'),
+    ('01', 'Aland', 'FI'),
+    ('02', 'South Karelia', 'FI'),
+    ('03', 'South Ostrobothnia', 'FI'),
+    ('04', 'South Savo', 'FI'),
+    ('05', 'Kainuu', 'FI'),
+    ('06', 'Kanta-Hame', 'FI'),
+    ('07', 'Central Ostrobothnia', 'FI'),
+    ('08', 'Central Finland', 'FI'),
+    ('09', 'Kymenlaakso', 'FI'),
+    ('10', 'Lapland', 'FI'),
+    ('11', 'Pirkanmaa', 'FI'),
+    ('12', 'Ostrobothnia', 'FI'),
+    ('13', 'North Karelia', 'FI'),
+    ('14', 'North Ostrobothnia', 'FI'),
+    ('15', 'North Savo', 'FI'),
+    ('16', 'Paijat-Hame', 'FI'),
+    ('17', 'Satakunta', 'FI'),
+    ('18', 'Uusimaa', 'FI'),
+    ('19', 'Southwest Finland', 'FI'),
+    ('OV', 'Overijssel', 'NL'),
+    ('GR', 'Groningen', 'NL'),
+    ('FR', 'Friesland', 'NL'),
+    ('FL', 'Flevoland', 'NL'),
+    ('NH', 'North Holland', 'NL'),
+    ('UT', 'Utrecht', 'NL'),
+    ('NB', 'North Brabant', 'NL'),
+    ('ZE', 'Zeeland', 'NL'),
+    ('LI', 'Limburg', 'NL'),
+    ('GE', 'Gelderland', 'NL'),
+    ('DR', 'Drenthe', 'NL'),
+    ('ZH', 'South Holland', 'NL'),
+    ('BE', 'Berlin', 'DE'),
+    ('HB', 'Free Hanseatic City of Bremen', 'DE'),
+    ('SL', 'Saarland', 'DE'),
+    ('NW', 'North Rhine-Westphalia', 'DE'),
+    ('RP', 'Rhineland-Palatinate', 'DE'),
+    ('BW', 'Baden-Wurttemberg', 'DE'),
+    ('NI', 'Lower Saxony', 'DE'),
+    ('HE', 'State of Hesse', 'DE'),
+    ('BY', 'Free State of Bavaria', 'DE'),
+    ('MV', 'Mecklenburg-Vorpommern', 'DE'),
+    ('HH', 'Free and Hanseatic City of Hamburg', 'DE'),
+    ('ST', 'Saxony-Anhalt', 'DE'),
+    ('SH', 'Schleswig-Holstein', 'DE'),
+    ('SN', 'Free State of Saxony', 'DE'),
+    ('TH', 'Free State of Thuringia', 'DE'),
+    ('BB', 'State of Brandenburg', 'DE'),
```

Comment:
lol, this is actually not something I've thought about so far, because we only have subdivisions for english speaking countries (well, apart from canada, kinda), but: Should we put these names in their original language, or in English translation? Because for Germany, these translations are pretty bad haha (I guess they're based on the legal names of the states, which were then translated into English?). But even in English, I would write them as follows:

```suggestion
    ('BE', 'Berlin', 'DE'),
    ('HB', 'Bremen', 'DE'),
    ('SL', 'Saarland', 'DE'),
    ('NW', 'North Rhine-Westphalia', 'DE'),
    ('RP', 'Rhineland-Palatinate', 'DE'),
    ('BW', 'Baden-Wurttemberg', 'DE'),
    ('NI', 'Lower Saxony', 'DE'),
    ('HE', 'Hessen', 'DE'),
    ('BY', 'Bavaria', 'DE'),
    ('MV', 'Mecklenburg-Vorpommern', 'DE'),
    ('HH', 'Hamburg', 'DE'),
    ('ST', 'Saxony-Anhalt', 'DE'),
    ('SH', 'Schleswig-Holstein', 'DE'),
    ('SN', 'Saxony', 'DE'),
    ('TH', 'Thuringia', 'DE'),
    ('BB', 'Brandenburg', 'DE'),
```

If we translate, let's check in with one player from each country to make sure we get the translations right (or just do these in native languages)

Reply:
sure!